private/bufpkg/bufmodule/uploader.go

```diff
@@ -65,6 +91,16 @@ type UploadOptions interface {
 	//
 	// Will always be present if CreateIfNotExist() is true.
 	CreateModuleVisibility() ModuleVisibility
+	// Tags returns unique and sorted set of tags to be added as labels.
```

Comment:
What are the differences with tags as opposed to labels? This comment implies they are the same.

My comment here is to make sure it is clear to future readers as to what the difference is, and why we have multiple fields.